rustls/src/manual/features.rs

```diff
@@ -1,47 +1,53 @@
 /*!
 
+Items where the support can be [extended or altered][CryptoProvider] are marked with
+an asterisk `*`.  The below list reflects the support provided with
+the default crate features.
+
+[CryptoProvider]: crate::crypto::CryptoProvider
+
 ## Current features
 
-* TLS1.2 and TLS1.3.
-* ECDSA, Ed25519 or RSA server authentication by clients.
-* ECDSA, Ed25519 or RSA server authentication by servers.
-* Forward secrecy using ECDHE; with curve25519, nistp256 or nistp384 curves.
-* AES128-GCM and AES256-GCM bulk encryption, with safe nonces.
-* ChaCha20-Poly1305 bulk encryption ([RFC7905](https://tools.ietf.org/html/rfc7905)).
-* ALPN support.
-* SNI support.
-* Tunable fragment size to make TLS messages match size of underlying transport.
-* Optional use of vectored IO to minimise system calls.
-* TLS1.2 session resumption.
-* TLS1.2 resumption via tickets ([RFC5077](https://tools.ietf.org/html/rfc5077)).
-* TLS1.3 resumption via tickets or session storage.
-* TLS1.3 0-RTT data for clients.
-* TLS1.3 0-RTT data for servers.
-* Server and optional client authentication.
-* Extended master secret support ([RFC7627](https://tools.ietf.org/html/rfc7627)).
-* Exporters ([RFC5705](https://tools.ietf.org/html/rfc5705)).
-* OCSP stapling by servers.
+* TLS1.2 and TLS1.3
```

Comment:
Tiny nit: it would be nice to split the removal of periods from the other changes here. Understand if you don't think that's worth it, though.

Reply:
have split that into its own commit